src/components/common/EthHashInfo/styles.module.css

```diff
@@ -27,6 +27,11 @@
   white-space: nowrap;
 }
 
+.addressRow svg {
+  height: 1.2em;
+  width: auto;
+}
+
```

Comment:
We can achieve the same result by removing the `fontSize="small"` prop from the `TokenExplorerLink` and `CopyButton`

Reply:
Good suggestion, it fixes it indeed. I grepped, and it looks like we have quite a bunch of places where this `fontSize="small"` is used on icons.

Reply:
I'm going to fix it only for copy and explorer in this PR to stay within the scope. Will open an issue to fix and check the rest.